src/NumericalAlgorithms/Interpolation/InterpolationTarget.hpp

```diff
@@ -59,8 +62,13 @@ namespace intrp {
 ///                                  are added by `initialize`.
 /// - post_interpolation_callback:   a struct with a type alias
 ///                                  `const_global_cache_tags` (listing tags
```

Comment:
Just throwing an idea out, not a change request. Maybe it would be better to have say 4 spaces indenting the `a struct with...` on the line after `post_interpolation_callback:`?

Reply:
I added another commit at the end that reformats the comments at the top of this file.